pkg/ccip/chainaccessor/ton_accessor.go

```diff
@@ -659,20 +659,16 @@ func (a *TONAccessor) GetChainFeePriceUpdate(ctx context.Context, selectors []cc
 			return nil, err
 		}
 
-		value, err := result.Cell(0)
-		if err != nil {
-			return nil, err
-		}
-
-		// HACK: we read the value as Timestamped since the binary layout is compatible, so that we match TimestampedBig (two values packed together)
-		var update feequoter.TimestampedPrice
-		if err := tlb.LoadFromCell(&update, value.BeginParse()); err != nil {
-			return nil, fmt.Errorf("failed to decode TimestampedPrice, potentially unsynced gobindings: %w", err)
-		}
+		// The plugin expects ExecutionGasPrice and DataAvailabilityGasPrice to be packed into a single big.Int
+		// value where DataAvailabilityGasPrice occupies the higher 112 bits and ExecutionGasPrice occupies the
+		// lower 112 bits. This allows DA and exec gas prices to be represented in a single value for L2 rollups.
+		// The below is a is a bitwise operation: (dataAvFeeUSD << 112) | executionFeeUSD
```

Comment:
I think you can technically pretend it's two `u112` values and parse it that way, at least that's what the original implementation I had assumed (so rather than reading one value and splitting it in two, you read two shorter values). Maybe the endianness is different?